src/Layout/redist/targets/RestoreLayout.targets

```diff
@@ -12,7 +12,7 @@
 
     <!-- Blob storage directories are not stabilized, so these must refer to a package that does not stabilize -->
     <!-- In unified build, the layout does match, so use the runtime package versions rather than the VS redist versions -->
-    <AspNetCoreBlobVersion>$(MicrosoftAspNetCoreAppRefInternalPackageVersion)</AspNetCoreBlobVersion>
+    <AspNetCoreBlobVersion>$(MicrosoftAspNetCoreAppRuntimePackageVersion)</AspNetCoreBlobVersion>
```

Comment:
Is this change related to this PR?

Reply:
Yes, it's necessary for this PR. Ultimately, this value comes from the `MicrosoftAspNetCoreAppRefPackageVersion` property which is correctly overridden by SB to the live version. Leaving this set to `MicrosoftAspNetCoreAppRefInternalPackageVersion` would cause a break because SB doesn't provide an overridden value for it and so it's left as its original hardcoded value from Versions.props which can't be resolved. In the case of UB, both `MicrosoftAspNetCoreAppRefInternalPackageVersion` and `MicrosoftAspNetCoreAppRuntimePackageVersion` have the same value.